### ISSUE TITLE
Продамус: можем ли мы поменять страницу на stepanov_me? Вместо stepanov-me как они сделали?

### ISSUE THREAD

**Issue by erulki** [MikStepv/MS#32]

**Comment by erulki** [comment]
да, можем. это стоит 1000р, они могут выслать на почту ссылку для оплаты.
я им написала, если мы согласны, то нужно мне ответным письмом отправить что да, мы согласны и написать итоговый вариант, как должен выглядеть домен

**Comment by MikStepv** [comment]
Поднимаем анкету и смотрим как там. Если это возможно, то просим изменить дефис на нижнее подчеркивание. 

Личный звонок

**Comment by erulki** [comment]
пытаюсь дозвониться, пока что трубки не берут

**Comment by erulki** [comment]
![Image](https://user-images.githubusercontent.com/122890135/219067126-84d6e351-4a16-4be0-b5e2-36c7fe9cc043.jpeg)
